dmcontent/govuk_frontend.py

```diff
@@ -215,8 +215,10 @@ def govuk_radios(
             params["classes"] += " govuk-radios--inline"
         else:
             params["classes"] = "govuk-radios--inline"
-
-    params["items"] = govuk_options(question.options, data.get(question.id))
+        options = [{"label": "Yes", "value": "True"}, {"label": "No", "value": "False"}]
+        params["items"] = govuk_options(options, f"{data.get(question.id)}")
```

Comment:
```suggestion
        params["items"] = govuk_options(options, data.get(question.id))
```

Reply:
Tried this initially, but it then fails in dm_utils - `govuk_options` is expecting this to be a string or dict of strings - `essentialRequirementsMet` on Brief Responses data property is:

`{'essentialRequirementsMet': True}`

If better to adjust the utils method, I can do that?

Reply:
Oh in that case I would suggest doing this: 

```suggestion
        params["items"] = govuk_options(options, str(data.get(question.id)))
```

It amounts to the same thing but is more explicit...

Or even more explicit would be 

```suggestion
        params["items"] = govuk_options(options, "True" if data.get(question.id) else "False")
```

but I'm fairly confident the string representation of true and false isn't changing any time soon....